content/Software and Downloads/Upload/Error-avrdude-when-uploading.md

```diff
@@ -2,58 +2,87 @@
+   ![The reset button on Arduino UNO.](img/uno-reset-button.png)
 
-6. Make sure that you have closed the serial monitor.
+4. Check port and cable connections:
 
-7. Press the reset button and see if the LED blinks. If the LED doesn't blink there is a high chance that the bootloader of your board is corrupted.
+   * Try disconnecting and reconnecting the board to the computer, and confirm that the power LED (usually labeled on the board "ON") is lit. On most boards, an orange LED will also blink when the board is plugged in. <!-- TODO: Simplify -->
 
-8. Make sure you have updated and installed all the necessary drivers for the board. If you already have the drivers, try reinstalling them.
+   * If possible, connect the board directly to a USB port. Avoid using a USB hub, or try a dif
 
-9. Try a complete uninstall (including deleting the [Arduino15 folder](https://support.arduino.cc/hc/en-us/articles/360018448279) of the Arduino IDE. Then reinstall the IDE using the latest version.
+   * Make sure you are using a data transfer USB cable (charge-only cables will not work). Try connecting your board with a different cable, or testing your cable with a different device.
 
-10. If your board is an Uno, Mega, or Nano board, you can do a [loopback test](https://support.arduino.cc/hc/en-us/articles/360020366520). This will rule out faulty hardware.
+   * Remove any unneeded USB devices that may be using the ports.
 
-11. If the loopback test passes, the problem may be due to a corrupted bootloader. In this case, burning a fresh bootloader may solve the issue. Keep in mind that this is an advanced procedure. Use the links below to get started.
+5. Remove any connections to digital pins **0(RX)** and **1(TX)**. Anything connected to these pins can interfere with the upload transfer.
 
-    * [General overview on bootloader burning](https://www.arduino.cc/en/Tutorial/BuiltInExamples/ArduinoISP)
-    * [Using two Arduino Mega boards](https://support.arduino.cc/hc/en-us/articles/360012048060)
-    * [Using two Arduino UNO boards](https://support.arduino.cc/hc/en-us/articles/360012048080)
+6. Press the reset button and see if the LED blinks. If the LED doesn't blink there is a high chance that the bootloader of your board is corrupted.
+
+7. Try a complete uninstall (including deleting the [Arduino15 folder](https://support.arduino.cc/hc/en-us/articles/360018448279) of the Arduino IDE. Then reinstall the IDE using the latest version.
+
+8. For some boards, you can do a [loopback test](https://support.arduino.cc/hc/en-us/articles/360020366520) to test the USB to serial chip. If the loopback test passes, or if you can't do a loopback test, the bootloader may be corrupted or missing. In this case, try [burning a fresh bootloader](https://docs.arduino.cc/built-in-examples/arduino-isp/ArduinoISP). The loopback test can be done with these boards:
```

Comment:
```suggestion
8. For some boards, you can do a [loopback test](https://support.arduino.cc/hc/en-us/articles/360020366520) to test the USB to serial chip. If the loopback test passes, the bootloader may be corrupted or missing. In this case, try [burning a fresh bootloader](https://docs.arduino.cc/built-in-examples/arduino-isp/ArduinoISP). The loopback test can be done with these boards:
```

This information is a bit confusing. In the [updated version of "how to do a loopback test"](https://github.com/arduino/help-center-content/pull/22/files?short_path=82742e1#diff-82742e1cd70d42fd21f29977c195455fdbe17d71461f5bd98374b6f1811531a1) says that a failed loopback test indicates that the USB to TTL serial adapter is damaged. 

Maybe we should only mention the passed loopback test case.

Reply:
I've removed this from the "general checklist", as it only is relevant for a limited set of boards and errors.

But see the changes to `avrdude: stk500_recv(): programmer is not responding`.